daemon/daemon.go

```diff
@@ -581,6 +588,20 @@ func (d *Daemon) installIptablesRules() error {
 		return err
 	}
 
+	// Mark all packets sourced from the host with a special marker so that
```

Comment:
Please be more specific here and state that we mark packets from *processes* running on the host.